Tests/js/views/services/assets/ez-roleserversideviewservice-tests.js

```diff
@@ -632,8 +633,61 @@ YUI.add('ez-roleserversideviewservice-tests', function (Y) {
         }
     });
 
+    refreshAssignmentsTabTest = new Y.Test.Case({
+        name: "eZ Role Server Side View Service refresh assignments tab test",
+
+        setUp: function () {
+            this.capi = new Mock();
+            this.app = new Mock();
+            this.roleId = 42;
+
+            this.service = new Y.eZ.RoleServerSideViewService({
+                capi: this.capi,
+                app: this.app,
+            });
+
+            Mock.expect(this.service, {
+                method: 'get',
+                args: [Mock.Value.String],
+                returns: this.app,
+            });
+            Mock.expect(this.service, {
+                method: 'load',
+                args: [Mock.Value.Function],
+            });
+            Mock.expect(this.app, {
+                method: 'get',
+                args: [Mock.Value.String],
+            });
+        },
+
+        tearDown: function () {
+            delete this.capi;
+            delete this.app;
+            delete this.service;
+        },
+
+        "Should refresh the page with tabId parameter added": function () {
+            var data = {
+                    roleId: this.roleId,
+                };
+
+            Mock.expect(this.app, {
+                method: 'set',
+                args: ['loading', true],
+            });
+            Mock.expect(this.app, {
+                method: 'navigateTo',
+                args: ['adminRole', {uri: 'pjax/role/view/' + this.roleId + '/ez-tabs-content'}],
+            });
+
+            this.service._refreshAssignmentsTab(data);
```

Comment:
You shouldn't call private/protected methods directly. To run it you should call the event that triggers it so probably something like: 
```javascript
this.service.fire('refreshAssignmentsTab', {...});
```